### PR TITLE
Assemble release team for 1.9

### PR DESCRIPTION
**Pull request by rimolive**

### CODE REVIEW

**Comment by rimolive** [comment]
@chensun @zijianjoy Can you approve?

**Comment by rimolive** [comment]
@DnPlas @annajung Maybe you can approve?

File: github-orgs/kubeflow/org.yaml [hunk 1]
```diff
@@ -966,7 +966,7 @@ orgs:
           release-managers:
             description: Kubeflow Release Team - Managers
             members:
-              - DnPlas
+              - rimolive
```
Comment (zijianjoy) [inline github-orgs/kubeflow/org.yaml:969]: Just want to make sure @DnPlas comment on the handover. And then I can approve.

**Review by DnPlas — APPROVED** [review]
Thanks @rimolive LGTM!

File: github-orgs/kubeflow/org.yaml (hunk 1 above)
Comment (DnPlas) [reply github-orgs/kubeflow/org.yaml:969]: +1, @rimolive will be the next release manager.

**Comment by zijianjoy** [comment]
/lgtm
/approve

Thank you @DnPlas for leading a successful Kubeflow release! And welcome @rimolive for leading the next one!

**Comment by rimolive** [comment]
Closes kubeflow/manifests#2605

**Pull request merged**